apps/judicial-system/backend/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -201,6 +202,14 @@ export class Defendant extends Model {
   @ApiPropertyOptional({ type: Boolean })
   isSentToPrisonAdmin?: boolean
 
+  @Column({
```

Comment:
Nice! I see you picked up on our tendancy to store these types of fields as strings in the database but deal with them as ENUM's in the code. We used to store them as enums in the db as well but the constant migrations when adding or removing fields became tedious so we decided on this route instead 👌

Reply:
💡 Ah okay I see, thanks for the context 🙏